package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xfc",
-  "version": "1.3.0",
+  "version": "1.3.2",
```

Comment:
Do the release in a separate PR and update release notes. This will be released with https://github.com/cerner/xfc/pull/18 and should probably be `1.4.0`

Reply:
I usually do two separate PRs, but I can combine them. hold on a second.